src/aurelia-binding.d.ts

```diff
@@ -115,8 +115,10 @@ export declare class EventManager {
    * @param targetEvent Name of event to subscribe.
    * @param callback Event listener callback.
    * @param delegate True to use event delegation mechanism.
+   * @param diposable True to return a disposable object with dispose() method instead of a function
    * @returns function wich removes event listener.
    */
+  addEventListener(target: Element, targetEvent: string, callback: (event: Event) => any, delegate: delegationStrategy, disposable: true): Disposable;
```

Comment:
Shouldn't the default for `disposable` be `false` in order to maintain backwards compatibility.

Reply:
It's only the typing, which tells we only return Disposable when and only when the last parameter is `true`, not just truthy. This is to strict the API and avoid breaking changes, as if someone used this API (doubt this  ) their 5th parameter needs to be `true` to have their code potentially break.

Reply:
Oh! Gotcha. I looked at it in isolation and didn't realize it was providing a signature specific for that code path. Cool!